### DISCUSSION TITLE
Public Transportation

### DISCUSSION THREAD

**Discussion by kelsangchandra** [microCOVID/microCOVID#1496] [General]
Hi, thank you so much for your work, this website has been extremely helpful for me. I was wondering if you could include a Public Transportation scenario such as BART in San Francisco. 
thanks!
Chandra

**Comment by apiology** [comment]
As someone who rides BART's sister system (DC Metrorail), I'm also keen on tracking this!  I've been using microCOVID's Environment option "A train with air filtration", but I'd recommend checking up on information from your system on their filtration equipment.

Perhaps it'd make sense as a scenario, especially if we had some kind of rational basis for trimming down the current long-ish list a bit that justifies a slot for this one in the result.
